docs/src/tutorials/Installation_and_configuration.md

```diff
@@ -0,0 +1,330 @@
+# Getting started with thin-edge.io on a Raspberry Pi 
+
+After following this tutorial you will have an overview of the installation and configuration of thin-edge.io. As an example, a Raspberry Pi is used. This tutorial explains in small steps to reach the goal of sending data to an IoT cloud platform (in this case, Cumulocity IoT) and performing some additional device management tasks.
+
+
+## Introduction
+
+thin-edge.io is an open-source project initiated by Software AG and partners to provide a cloud-agnostic edge framework. It is much more generic than the device management agent, so it can connect to multiple IoT cloud platforms, and it allows flexible logic executed on the device. It is optimized for a very small footprint and high performance.
```

Comment:
I'm not convinced about citing SoftwareAG and the device management agent. This would be the very first such reference in the whole documentation. This is not even said in the vision. The device management agent might unknown outside of SoftwareAG.